src/psm/test/gcd_no_vsrc.tcl

```diff
@@ -4,5 +4,6 @@ read_lef Nangate45/Nangate45.lef
 read_def Nangate45_data/gcd.def
 read_liberty Nangate45/Nangate45_typ.lib
 read_sdc Nangate45_data/gcd.sdc
-set_pdnsim_net_voltage -net VDD -voltage 1.1
+set_pdnsim_net_voltage -net VDD -voltage 1.5
```

Comment:
Why the change?

Reply:
because the default value is 1.1V, so setting it to 1.1 doesn't actually prove that this is working. I just picked a different number so I could see the value change from the default.